docs/src/user/cli/info.rst

```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
 
 The last section contains information about the best trial so far, providing its
 hyperparameter values and the corresponding objective.
+
+The ``--name`` argument
+~~~~~~~~~~~~~~~~~~~~~~~
+To get information on an experiment, you need to call `info` with the `--name` or `-n` argument like
+shown in the previous example. This will fetch the latest version of the experiment with that name
+inside the database and display its content.
+
+The ``--version`` argument
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+To specify which version of an experiment you wish to observe, you can use the `--version` argument.
+If provided, this will fetch the experiment with a version number corresponding to that version
+instead of fetching the latest one. Note that the `--version` argument cannot be used alone and that
```

Comment:
For info it can be used alone because there is no `--name` argument.

Reply:
There is.